<launch> 
	<node pkg="landing" type="simple_landing_node" name="simple_landing_node" output="screen"/> 
	<node pkg="landing" type="keyboard.py" name="keyboard"  />
	<node pkg="landing" type="track640.py" name="track">
          <param name="is_sim" type="int" value="0"/>
  </node>
</launch>

